src/System/Configuration/DataSets/ConfigurationDataSetLoader.cs

```diff
@@ -44,11 +50,13 @@ public abstract class ConfigurationDataSetLoader<T> : ConfigurationDataSetLoader
 		/// </summary>
 		/// <param name="cache">The cache object.</param>
 		/// <param name="resourceMonitor">Resource Watcher instance</param>
+		/// <param name="dataSetFactory">The data set factory.</param>
 		/// <param name="dataSetOverride">The data set override.</param>
-		protected ConfigurationDataSetLoader(ICache cache, IResourceMonitor resourceMonitor, T dataSetOverride = null)
+		protected ConfigurationDataSetLoader(ICache cache, IResourceMonitor resourceMonitor, IDataSetFactory<T> dataSetFactory, T dataSetOverride = null)
 		{
 			Cache = Code.ExpectsArgument(cache, nameof(cache), TaggingUtilities.ReserveTag(0x238208d9 /* tag_9669z */));
 			m_resourceMonitor = Code.ExpectsArgument(resourceMonitor, nameof(resourceMonitor), TaggingUtilities.ReserveTag(0x23821000 /* tag_967aa */));
+			DataSetFactory = Code.ExpectsArgument(dataSetFactory, nameof(dataSetFactory), TaggingUtilities.ReserveTag(0));
```

Comment:
That shouldn't be possible because of the NuGet dependencies, or am I missing something?

---
In reply to: [306890022](https://github.com/microsoft/Omex/pull/66#discussion_r306890022) [](ancestors = 306890022)

Reply:
Discussed offline, the NuGet package dependencies will handle this.

---
In reply to: [307200376](https://github.com/microsoft/Omex/pull/66#discussion_r307200376) [](ancestors = 307200376,306890022)